### PR TITLE
coap-gw: update config

### PR DESCRIPTION
**Pull request by jkralik**

### CODE REVIEW

File: content/en/configuration/coap-gateway.md [hunk 1]
```diff
@@ -58,11 +58,12 @@ OAuth2.0 Client is used to issuing an authorization code used by the Onboarding
 
 | Property | Type | Description | Default |
 | ---------- | -------- | -------------- | ------- |
-| `api.coap.authorization.ownerClaim` | string | `Claim used to identify owner of the device.` | `"sub"` |
+| `api.coap.authorization.ownerClaim` | string | `Claim used to identify owner of the device. Do not use api.coap.authorization.ownerClaim with "sub" value for api.coap.authorization.providers[].grantType with clientCredentials. Otherwise, all devices will be under the same user.` | `"sub"` |
```
Comment (ondrejtomcik) [inline content/en/configuration/coap-gateway.md:61]: We need to prevent this combination programatically. Not documenting it.

Comment (jkralik) [reply content/en/configuration/coap-gateway.md:61]: Fixed in https://github.com/plgd-dev/hub/pull/669

Comment (ondrejtomcik) [reply content/en/configuration/coap-gateway.md]: Please use the formulation from hub MR.

Comment (jkralik) [reply content/en/configuration/coap-gateway.md]: fixed

**Review by ondrejtomcik — APPROVED** [review]

**Pull request merged**